### ISSUE TITLE
Unable to build dockerfile on windows

### ISSUE THREAD

**Issue by grizzm0** [moby/moby#30472]
**Description**

**Steps to reproduce the issue:**
1. Create a new Dockerfile with anything in it (FROM should be enough)
2. `docker build -t foobar .`

**Describe the results you received:**
The build gets stuck on "Building foobar"

**Describe the results you expected:**
I'd expect the dockerfile to build.

**Additional information you deem important (e.g. issue happens only occasionally):**
Worked fine a few weeks ago when I built the same setup. Updated to 1.13 a few days ago, might be related.

**Output of `docker version`:**

```
Client:
 Version:      1.13.0
 API version:  1.25
 Go version:   go1.7.3
 Git commit:   49bf474
 Built:        Tue Jan 17 21:19:34 2017
 OS/Arch:      windows/amd64

Server:
 Version:      1.13.0
 API version:  1.25 (minimum version 1.12)
 Go version:   go1.7.3
 Git commit:   49bf474
 Built:        Wed Jan 18 16:20:26 2017
 OS/Arch:      linux/amd64
 Experimental: true
```

**Output of `docker info`:**

```
Containers: 0
 Running: 0
 Paused: 0
 Stopped: 0
Images: 3
Server Version: 1.13.0
Storage Driver: overlay2
 Backing Filesystem: extfs
 Supports d_type: true
 Native Overlay Diff: true
Logging Driver: json-file
Cgroup Driver: cgroupfs
Plugins:
 Volume: local
 Network: bridge host ipvlan macvlan null overlay
Swarm: inactive
Runtimes: runc
Default Runtime: runc
Init Binary: docker-init
containerd version: 03e5862ec0d8d3b3f750e19fca3ee367e13c090e
runc version: 2f7393a47307a16f8cee44a37b262e8b81021e3e
init version: 949e6fa
Security Options:
 seccomp
  Profile: default
Kernel Version: 4.9.5-moby
Operating System: Alpine Linux v3.5
OSType: linux
Architecture: x86_64
CPUs: 6
Total Memory: 9.725 GiB
Name: moby
ID: DJZI:F2KQ:67S7:TQFU:U3BV:AZNI:O6Z5:AVCI:QJAY:3ZTA:QWIX:E4S4
Docker Root Dir: /var/lib/docker
Debug Mode (client): false
Debug Mode (server): true
 File Descriptors: 13
 Goroutines: 22
 System Time: 2017-01-26T16:23:44.4794966Z
 EventsListeners: 0
Registry: https://index.docker.io/v1/
Experimental: true
Insecure Registries:
 127.0.0.0/8
Live Restore Enabled: false
```

**Additional environment details (AWS, VirtualBox, physical, etc.):**

**Comment by jhowardmsft** [comment]
To confirm this works from a remote Linux client - just hangs on a remote Windows client?

**Comment by grizzm0** [comment]
I don't have the windows install anymore. But now that I started using docker for mac on a large webdev project I noticed that the initial "Building x" takes a while. After some goolging I figured out it was due to docker tarballing the context dir.

The issue on windows was probably related to this, just that the windows folder had lots and lots of files. Would be solved by .dockerignore.

Closing as I don't think this was a real issue.
